### PR TITLE
Added custom item documentation

### PR DESCRIPTION
**Pull request by ImDaBigBoss**
The link I use line 121 of the doc file won't work until custom items are merged into master.

### CODE REVIEW

File: _docs/geyser/custom-items.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+        }
+    }
+    ```
+
+## Geyser extensions
+
+### Extending a vanilla item
+
+1. In your extension config file, you need to set: `startup-phase: "PRE_INITIALIZE"` so that the pre init event can be caught, and your items can be registered.
```
Comment (RednedEpic) [inline _docs/geyser/custom-items.md:80]: Config file or the `extension.yml`?

File: _docs/geyser/custom-items.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+```
+5. Then, in your pre init event, you can register your item:
+```java
+@Subscribe
+public void onGeyserPreInitializeEvent(GeyserPreInitializeEvent event) {
+    this.geyserApi().customItemManager().registerCustomItem("minecraft:JAVA_ITEM", data);
+}
+```
+
+### Non vanilla items with Geyser extensions (for example to use with Fabric)
```
Comment (RednedEpic) [inline _docs/geyser/custom-items.md:111]:
```suggestion
### Non vanilla (modded) items with Geyser extensions (for example to use with Fabric)
```

File: _docs/geyser/custom-items.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+1. In your extension config file, you need to set: `startup-phase: "PRE_INITIALIZE"` so that the pre init event can be caught, and your items can be registered.
```
Comment (RednedEpic) [inline _docs/geyser/custom-items.md:113]: See earlier comment.

**Review by RednedEpic — DISMISSED** [review]

File: _docs/geyser/custom-items.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
+---
+layout: page
+title: Page title
```
Comment (Chew) [inline _docs/geyser/custom-items.md:3]: Needs a page title

**Review by Camotoy — APPROVED** [review]

**Pull request merged**